apps/wpcom-block-editor/src/wpcom/features/switch-to-classic-notice.js

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable import/no-extraneous-dependencies */
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { registerPlugin } from '@wordpress/plugins';
+import { PluginDocumentSettingPanel } from '@wordpress/edit-post';
+import url from 'url';
+/* eslint-enable import/no-extraneous-dependencies */
+
+const parsedEditorUrl = url.parse( window.location.href, true );
+
+const classicEditorUrl = `${ parsedEditorUrl.protocol }//${ parsedEditorUrl.hostname }${ parsedEditorUrl.pathname }?post=${ parsedEditorUrl.query.post }&action=${ parsedEditorUrl.query.action }&set-editor=classic`;
```

Comment:
this code runs outside calypso, so can't use the getWpAdminClassicEditorRedirectionUrl selector, but this approach seems to work ... haven't tested with atomic or jetpack site.